defaults/main.yml

```diff
@@ -4,7 +4,9 @@ zookeeper_playbook_version: "0.9.2"
 zookeeper_version: 3.4.6
 zookeeper_url: http://www.us.apache.org/dist/zookeeper/zookeeper-{{zookeeper_version}}/zookeeper-{{zookeeper_version}}.tar.gz
 
+zookeeper_debian_apt_install: false
```

Comment:
Note: 2 things we could change here:
* set default to 'true'  (assuming we want to keep offering also the original debian apt install)
* rename variable to zookeeper_tarball_install and set 'true' (with the idea we could later also provide a zookeeper yum/rpm setup on redhat)